.github/workflows/ci.yml

```diff
@@ -209,9 +209,12 @@ jobs:
       - name: Install
         run: |
           yarn install
-      - name: Downgrade Jest for node <= 10
+      - name: Downgrade Jest and Babel for node <= 10
```

Comment:
Why do we need to do this? Don't we build compile on a modern Node.js version, and _then_ downgrade Node.js to run Jest?

Reply:
Jest will invoke `@babel/core` to transpile the test code so that it can run on Node.js 6 - 10, which is not supported by Babel 8. If we pre compile the test code, we don't have to downgrade `@babel/core` here.